src/main/java/decodex/parser/Parser.java

```diff
@@ -19,47 +19,40 @@
  */
 public class Parser {
 
-    /**
-     * Specifies the index of the tokens where specific parameters can be found.
-     */
+    // Specifies the index of the tokens where specific parameters can be found.
     private static final int COMMAND_INDEX = 0;
 
-    /**
-     * Specifies the starting index where the arguments can be found.
-     */
+    // Specifies the starting index where the arguments can be found.
     protected static final int STARTING_ARGUMENTS_INDEX = 1;
 
-    /**
-     * Specifies the token used to split the user input by.
-     */
+    // Specifies the tokens used to split the user input by.
     private static final String SPLIT_REGEX = "\\s+";
+    private static final int SPLIT_LIMIT_VALUE = -1;
 
-    /**
-     * Specifies the valid length of the tokens and used to check validity of tokens.
-     */
+    // Specifies the separator for joining a split array.
+    private static final String SPACE_SEPARATOR = " ";
```

Comment:
Perhaps say `Specifies the separator used for reconstructing the string from a list of tokens`?